### DISCUSSION TITLE
Translation (Portuguese Portugal)

### DISCUSSION THREAD

**Discussion by PoetaGA** [DineshSolanki/FoliCon#159] [General]
Hi,

I have translated to Portuguese (Portugal). I have the file. Can give push access to upload the file, or you prefere that i send by email?

Thanks for your work.

PoetaGA

**Comment by DineshSolanki** [comment]
Hi @PoetaGA thanks for this.
Would you be able to fork FoliCon repository and then create a branch from master, and raise a pull request.

That's the usual way, however if you're not comfortable with that, feel free to leave the file here, if you can upload.

**Comment by PoetaGA** [comment]
Hi @DineshSolanki ,

Done. Thanks.

PoetaGA

**Reply by DineshSolanki** [reply]
I think you only committed in your fork,  but no worries, I have raised the Pull request from your branch to here. 
https://github.com/DineshSolanki/FoliCon/pull/160

**Reply by DineshSolanki** [reply]
released in FoliCon v12.0 @PoetaGA, please download and let me know if there's any issue with it.

**Reply by PoetaGA** [reply]
Works. I'm going to improve the Translation later. Thanks.
